template-parts/events/browser/weekday-weather.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace GMUCF\Theme\TemplateParts\Events\Browser\WeekdayWeather;
+use GMUCF\Theme\Includes\Weather;
+
+
+$weather = Weather\get_weather('weather-extended');
+if (!empty($weather)) {
+?>
+<div id="weekday-weather" class="weather">
+	<div class="day">
+		<div class="day-image">
+			<h2>Today</h2>
+			<img src="<?php echo bloginfo('stylesheet_directory'); ?>/static/img/weather/<?php echo $weather['day1']['imgCode']; ?>.png" />
```

Comment:
We should consider creating functions that will spit out the appropriate string here so we can reduce the amount of code that needs to be on each line here. Would improve readability, and I don't really think it would hurt from an understanding the code point of view.

Reply:
Issue added.